src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -287,10 +287,12 @@ export default {
 				return PlainText
 			} else if (messagePartType === 'user') {
 				return Mention
-			} else if (messagePartType === 'call') {
-				return Mention
-			} else if (messagePartType === 'guest') {
-				return Mention
+			// FIXME Reenable once the UserBubble allows non-users
+			// FIXME Ref: https://github.com/nextcloud/nextcloud-vue/issues/796
+			// } else if (messagePartType === 'call') {
+			// return Mention
+			// } else if (messagePartType === 'guest') {
+			// return Mention
```

Comment:
This prevents the error in the console about `user` property being required when the mentions are rendered in the message list. But it does not prevent the error from being shown when the mentions are added in the new message input.

I would still enable the mentions already, as the error is shown only in development mode, it is just a Vue warning and nothing that makes the component fail, and the fix in nextcloud-vue should be just a few days away, but in the meantime the mentions can be used without problems.